sys-devel/binutils/binutils-2.26.1_2016_07_22.recipe

```diff
@@ -94,17 +95,25 @@ BUILD()
 	# Touch all *.info files, as newer texinfos don't like their format
 	(cd $sourceDir; find . -name \*.info | xargs touch)
 
-	(cd $sourceDir; find . -name \*.y | sed 's/.y$/.c/g' | xargs touch)
-    (cd $sourceDir; find . -name \*.y | sed 's/.y$/.h/g' | xargs touch)
+	# Touch all *.y files, ensure they get generated; without this, the
+	# PE targets would fail to link
+	(cd $sourceDir; find . -name \*.y | xargs touch)
 
 	mkdir -p $objectsDir
 	cd $objectsDir
 
+	binUtilsTargets=$effectiveTargetMachineTriple
+
+	if [ $effectiveTargetArchitecture == x86_64 ]; then
```

Comment:
@jessicah, I think it would be (slightly) better to use
```
[ $effectiveTargetArchitecture = x86_64 ]
```
instead of
```
[ $effectiveTargetArchitecture == x86_64 ]
```
because **`==`** only works with bash, while **`=`** works with both bash ans sh.